#Hk row number and column number
Hkr: 3
Hkc: 6

#Hk 1st row
Hk00: 1
Hk01: 0
Hk02: 0
Hk03: 0
Hk04: 0
Hk05: 0

#Hk 2nd row
Hk10: 0
Hk11: 1
Hk12: 0
Hk13: 0
Hk14: 0
Hk15: 0

#Hk 3rd row
Hk20: 0
Hk21: 0
Hk22: 1
Hk23: 0
Hk24: 0
Hk25: 0



#Fk row and column number
Fkr: 6
Fkc: 6

#Fk 1st row
Fk00: 1
Fk01: 0
Fk02: 0
Fk03: 0.01
Fk04: 0
Fk05: 0

#Fk 2nd row
Fk10: 0
Fk11: 1
Fk12: 0
Fk13: 0
Fk14: 0.01
Fk15: 0

#Fk 3rd row
Fk20: 0
Fk21: 0
Fk22: 1
Fk23: 0
Fk24: 0
Fk25: 0.01

#Fk 4th row
Fk30: 0
Fk31: 0
Fk32: 0
Fk33: 1
Fk34: 0
Fk35: 0

#Fk 5th row
Fk40: 0
Fk41: 0
Fk42: 0
Fk43: 0
Fk44: 1
Fk45: 0

#Fk 6th row
Fk50: 0
Fk51: 0
Fk52: 0
Fk53: 0
Fk54: 0
Fk55: 1


#Qk row and column number
Qkr: 6
Qkc: 6

#Qk 1st row
Qk00: 12
Qk01: 0
Qk02: 0
Qk03: 0
Qk04: 0
Qk05: 0

#Qk 2nd row
Qk10: 0
Qk11: 12
Qk12: 0
Qk13: 0
Qk14: 0
Qk15: 0

#Qk 3rd row
Qk20: 0
Qk21: 0
Qk22: 12
Qk23: 0
Qk24: 0
Qk25: 0

#Qk 4th row
Qk30: 0
Qk31: 0
Qk32: 0
Qk33: 12
Qk34: 0
Qk35: 0

#Qk 5th row
Qk40: 0
Qk41: 0
Qk42: 0
Qk43: 0
Qk44: 12
Qk45: 0

#Qk 6th row
Qk50: 0
Qk51: 0
Qk52: 0
Qk53: 0
Qk54: 0
Qk55: 12



#Rk row number and column number
Rkr: 3
Rkc: 3

#Rk 1st row
Rk00: 16
Rk01: 0
Rk02: 0

#Rk 2nd row
Rk10: 0
Rk11: 16
Rk12: 0

#Rk 3rd row
Rk20: 0
Rk21: 0
Rk22: 16



#Bk row and column number
Bkr: 6
Bkc: 3

#Bk 1st row
Bk00: 0.00005
Bk01: 0
Bk02: 0

#Bk 2nd row
Bk10: 0
Bk11: 0.00005
Bk12: 0

#Bk 3rd row
Bk20: 0
Bk21: 0
Bk22: 0.00005

#Bk 4th row
Bk30: 0.1
Bk31: 0
Bk32: 0

#Bk 5th row
Bk40: 0
Bk41: 0.1
Bk42: 0

#Bk 6th row
Bk50: 0
Bk51: 0
Bk52: 0.1


#Covariance
Cov_order: 6 

#Xk, Zk, Uk, Kgain
Xk: 6
Zk: 3
Uk: 3
Kgainr: 6
Kgainc: 3
